maxim/tests/test_openai.py

```
@@ -194,7 +194,7 @@ def setUp(self):
         # This is a hack to ensure that the Maxim instance is not cached
         if hasattr(Maxim, "_instance"):
             delattr(Maxim, "_instance")
-        self.logger = Maxim().logger()
+        self.logger = Maxim({"base_url": baseUrl}).logger()
 
     def test_chat_completions(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_chat_completions`

Add return type annotation: `None`

(ANN201)